src/js/module/toolbox/sidebar.js

```diff
@@ -1,58 +1,97 @@
-import underscore from 'underscore';
+import { chain, range, map } from 'underscore';
 import jquery from 'jquery';
-import { View } from 'core';
-import ToolboxViewObject from './object';
-import ToolboxCollection from './collection';
-import Const from 'const';
+import { View, Collection } from 'core';
+import ToolboxModel from '../plant/model';
```

Comment:
Yep, thats wrong. The issue is described in #134. I don't want to change it in this PR...